ampd/src/tests/config_template.toml

```diff
@@ -58,6 +58,27 @@ rpc_url = 'http://127.0.0.1/'
 secs = 3
 nanos = 0
 
+[[handlers]]
+type = 'SolanaMsgVerifier'
+cosmwasm_contract = 'axelar1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqecnww6'
+max_tx_cache_entries = 6
+chain_name = 'solana'
+chain_rpc_url = 'http://127.0.0.1/'
+
+[handlers.rpc_timeout]
+secs = 3
+nanos = 0
+
+[[handlers]]
+type = 'SolanaVerifierSetVerifier'
+cosmwasm_contract = 'axelar1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqecnww6'
```

Comment:
What's this, and what would be the correct address ?

Reply:
That probably is supposed to point to the Voting Verifier contract [(which we init with this xtask here)](https://github.com/eigerco/solana-axelar-internal/blob/be027f98b4e31d1e4a5fc3818c40e0352abb4c53/solana/xtask/src/cli/cmd/cosmwasm.rs#L120-L154). The idea is that ampd is supposed to send tx status updates into that cosmwasm contract about which events are confirmed and which are not

Reply:
The "conclusive" version of the contract is not yet deployed / initialised but I think we're gonna stick with the default verifier contract provided by Axelar

Reply:
Yes sorry with the last refactor this [changed a bit](https://github.com/eigerco/axelar-amplifier/blob/2fe806690dcf7905149f943f71213069c6933a38/ampd/src/handlers/solana_verify_msg.rs#L78-L86) and i was mixing things.